drone/src/drone/agent/mod.rs

```diff
@@ -136,18 +139,11 @@ async fn ready_loop(nc: TypedNats, drone_id: DroneId, cluster: String) -> Result
 pub async fn run_agent(agent_opts: AgentOptions) -> Result<()> {
     let nats = agent_opts.nats.connection().await?;
 
-    // Ensure that status stream exists.
-    nats.add_jetstream_stream(
-        &BackendStateMessage::stream_name(),
-        BackendStateMessage::wildcard_subject(),
-    )
-    .await?;
```

Comment:
I was thinking we would still need this? Isn't there a chance that the jetstream stream won't exist before the drone tries to publish to it?